packages/pass-style/src/types.d.ts

```diff
@@ -22,7 +22,11 @@ export type PrimitiveStyle =
   | 'string'
   | 'symbol';
 
-export type ContainerStyle = 'copyRecord' | 'copyArray' | 'tagged';
+export type ContainerStyle =
+  | 'copyRecord'
+  | 'copyArray'
+  | 'byteArray'
```

Comment:
https://endojs.github.io/endo/types/_endo_pass_style.Passable.html documents that Containers aggregate other Passables, so I don't think it should include "byteArray", which should instead be either a special case or (if we're taking a data model perspective) a primitive (similar to "string"). Which reminds me that packages/patterns/README.md also needs updating.

Reply:
I just read patterns/README.md carefully, and did not spot the flaw you have in mind. Is this indeed the document you have in mind? What needs updating?

Reply:
> Ah, no change is needed now that we acknowledge ByteArray as primitive

Resolving.